blindsign/blindrsa/brsa_test.go

```diff
@@ -20,7 +19,7 @@ import (
 )
 
 // 2048-bit RSA private key
-var testPrivateKey = testingKey(`
+const testPrivateKey = `
 -----BEGIN RSA TESTING KEY-----
```

Comment:
nits: you might want to use a test RSA key from RFC 9500 https://github.com/thibmeu/rfc9500?tab=readme-ov-file#rsa-keys

Reply:
good idea.